client/spotify-gdpr.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Spotify GDPR Analysis</title>
+    <base target="_blank">
+    <link rel="stylesheet" href="css/base.css">
+    <link rel="stylesheet" href="css/spotify-gdpr.css">
+  </head>
+  <body>
+    <main>
+      <h1 id="title">
+        <span class="highlight-primary-1">Spotify GDPR Analysis</span>
+      </h1>
+      <span class="subtitle">See how you've changed</span>
```

Comment:
if its not too inconvenient, could you add a screenshot in PR description? No need to do so now cause you showed it to us before 👍